src/components/PanelForm/PanelForm.tsx

```diff
@@ -18,52 +23,80 @@ const PanelForm = <F extends FormikValues>({
   error,
   isEditing,
   isSaving,
+  subForms,
   ...props
 }: Props<F>) => {
+  const [view, setView] = useState<string | null>();
   // This component does not use the Panel component as Vanilla requires a strict
   // element hierarchy that can't be achieved using the normal components in
   // conjunction with portals.
   return (
     <>
       <div className="p-panel__header">
-        <h4 className="p-panel__title">
-          {isEditing ? `Edit ${entity}` : `Create ${entity}`}
-        </h4>
+        <div className="p-panel__title">
+          <PanelFormNavigation
+            panelEntity={entity}
+            setView={setView}
+            view={view}
+          />
+        </div>
       </div>
       <div className="p-panel__content">
-        {error ? (
-          <Row>
-            <Col size={12}>
-              <Notification severity={NotificationSeverity.NEGATIVE}>
-                {error}
-              </Notification>
-            </Col>
-          </Row>
-        ) : null}
-        <Formik<F> {...props}>
-          <Form>
+        {view
```

Comment:
These sub-forms are the secondary panel views e.g. adding entitlements to a role or group.